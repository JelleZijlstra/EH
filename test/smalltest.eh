#!/usr/bin/ehi
printvar: argc
