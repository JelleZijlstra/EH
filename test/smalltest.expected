%%stdout%%
@int 1
%%stderr%%
